src/cms-content/learn/learn-case-studies.json

```diff
@@ -104,7 +104,7 @@
ents county governments in the United States. Serving nearly 40,000 county-elected officials and 3.6 million county employees, NACo helps make counties across America healthy, safe, and vibrant. \n\nTo achieve its mission, NACo engages with members, nurtures county leadership skills and knowledge networks, and supplies county officials with strong, evidence-based resources. Since the onset of COVID, NACo has deployed tools and resources to help county officials track their own local COVID situation and navigate reopening decisions.\n\n## **What** is the challenge?\n\nNACo’s mission is to strengthen America’s counties, but counties across the United States vary greatly in terms of size and resources; they range from New York City, home to 1.6 million Manhattanites, to rural counties in the West with fewer than a 1,000 residents. For this reason, when it comes to COVID reporting and data analysis, county officials have widely varying resources and needs, and NACo cannot offer one-size-fits-all solutions.\n\nIn order to support its broad range of members, NACo needs a diverse set of tools and resources to encourage data-backed decisions amidst COVID. To combat the pandemic, county officials have to make many important policy decisions, including guidance on mask-wearing, social gatherings, and local business openings, and how to allocate health resources. There is an overwhelming amount of publicly available data that is inconsistent and contradictory, making these decisions all the more difficult, particularly since every county has experienced COVID differently.\n\nNACo’s county resources and tools, then, need to be robust and evidence-based, but also relevant for hyper-local decision making.\n\n## **How** is NACo responding to COVID?\n\nCovid Act Now (CAN) was able to fill the need for a powerful, localized tool with its **[COVID Response Simulator (CRS)](https://covidactnow.org/tools#covid-response-simulator)**. \n\nThe CRS allows users to view the potential impact of [non-pharmaceutical interventions](/glossary#npi) (NPIs), such as mask-wearing and closing schools, on COVID statistics like case numbers and hospitalizations. It includes CAN’s powerful epidemiology model and data pulled directly from official state and county sources, but all of the inputs and assumptions are modifiable and can be changed to reflect local realities. County officials can use this tool to predict the effects of different policy interventions and make informed decisions on how to manage COVID for their county’s specific circumstances.\n\n![Image of Covid Act Now's Covid Response Simulator, showing the location inputs, model assumptions, and graphs](/images_cms/naco-screenshot.png \"COVID Response Simulator\")\n\n> All of the inputs and assumptions are modifiable and can be changed to reflect local realities\n\nNACo features CAN’s CRS through its twice-weekly COVID update e-blasts to 40,000 county officials, and it has invited CAN to join its “Reopening Town Hall.”",
```

Comment:
Did this change or just the image?

Reply:
no, it changed the CRS link from resources to tools

![Screen Shot 2020-11-17 at 6 18 19 PM](https://user-images.githubusercontent.com/44076375/99462296-5bfdd500-2901-11eb-97d4-f0b6cabca24c.png)